.github/workflows/workflows_validator.yml

```diff
@@ -6,9 +6,28 @@ on:
       - workflows/*/input_templates/*.json
   workflow_dispatch:
 
+permissions:
+  id-token: write
+  contents: write
+
+env:
+  AWS_REGION: us-east-1
+  AWS_ROLE: arn:aws:iam::337532070941:role/google-aws-sync-aws-role
+
 jobs:
-  start-run:
-    runs-on: ubuntu-20.04
+  validate-input-templates:
+    runs-on: ubuntu-latest
     steps:
-      - run: |
-            echo "THIS ACTION STILL DOES NOTHING"
+    - uses: actions/checkout@v4
+
+    - name: Install dependencies
+      run: pip install boto3
+
+    - name: Configure AWS credentials
```

Comment:
All resources should be public, we shouldn't need to authenticate (+ better not to expose the account number in a public repo).

Reply:
you're right but if I remove this I get "S3 path does not exist" error on all the files. Do you know why?

Reply:
I think you need the equivalent of `--no-sign-request` cli flag, I tested it from command line and it works.

Reply:
this one did the magic:
`s3_client = boto3.client('s3', config=Config(signature_version=UNSIGNED))`

thanks!